### PR TITLE
Redo docs

### PR DESCRIPTION
**Pull request by jmcd**
Work in progress as a pull request for visibility

### CODE REVIEW

File: environment.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+|-|-|-|
+| Live    | IdentityServer  | TBD |
+| Live    | DataServer      | TBD |
+| Live    | DeveloperPortal | TBD |
+| Sandbox | IdentityServer  | TBD |
+| Sandbox | DataServer      | TBD |
+| Sandbox | DeveloperPortal | TBD |
+| DHI     | IdentityServer  | https://identity.dhi-dse.scot/ |
+| DHI     | DataServer      | https://data.dhi-dse.scot/ |
+| DHI     | DeveloperPortal | https://dhi-lenus-prod-portal.azurewebsites.net |
```
Comment (lenus-health) [inline environment.md]: Can this be changed to portal.dhi-dse.scot?

Comment (lynchjames) [inline environment.md]: Can this be changed to portal.dhi-dse.scot?

File: concepts.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- heart-rate of a user at an instant
+- number of steps a user took between between two instants
+- nutritional values of food a user consumed on a day
+- sleep quality of a user on a night
+
+## DataServer
+
+The _DataServer_ stores Samples.
+
+The DataServer is used by developers' Clients to read and write Samples via an API. API calls must be authorized to succeed. Authorization is a based on user granted permission for the Client making a call.
```
[the remark this answers is not in the document]
Comment (lynchjames) [reply concepts.md]: I would vote for using "client application" consistently.

File: api/authorization.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+## Access Token
+
+An access token represents a Client's permission to act on behalf of a user. To aquire an access token, a Client must use the standard OpenID Connect mechanism built into IdentityServer to authenticate the user.
+
+A Client will need:
+
+* Functionality to complete the OpenID Connect flow - likely this is provided by a third-party library appropriate for the platform that the Client runs on.
+* Client details: Client Id, Grant Type, Scopes, any Secret set
+
+The IdentityServer supports endpoint discovery via a discovery document at
```
[the remark this answers is not in the document]
Comment (jmcd) [reply api/authorization.md]: No - see concepts.md

File: api/dataserver/core_resources.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
+# Core Resorces
+
+## SampleData
+
+The _SampleData_ object represents a single Sample in the DataServer. SampleDatas are the units that a Client uses to create Samples, and are the unit that are returned from querying the DataServer.
```
[the remark this answers is not in the document]
Comment (jmcd) [reply api/dataserver/core_resources.md]: The docs will use the string "Client" until we take a decision

File: api/dataserver/core_resources.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+| udi               | UDI device identifier |
+| btn               | Bluetooth name |
+
+#### Subject
+
+The Subject of a Sample is the creator and owner, i.e. the person the Sample is about.
+
+When creating a Sample, the SampleData's Subject attribute is ignored – the identity of the owner is determined by authorization associated with the API call.
+
+When a Client queries the DataServer on behalf of an Agent, the Subject value represents the unique reationship between the Agent and owner – the actual user identifer is hidden.
```
[the remark this answers is not in the document]
Comment (jmcd) [reply api/dataserver/core_resources.md]: It's not

File: api/dataserver/samples/category_samples.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# CategorySamples
```
[the remark this answers is not in the document]
Comment (jmcd) [reply api/dataserver/samples/category_samples.md]: Added to the corresponding index.md

File: api/dataserver/samples/index.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+# Samples
+
+A _Sample_ is a represents a piece health data for a user for some time.
+
+The time of a Sample is defined by a start and end date-time. If the start and end values are the same, the Sample represents an instant in time.
```
[the remark this answers is not in the document]
Comment (jmcd) [reply api/dataserver/samples/index.md]: instant

**Pull request merged**